src/peergos/server/tests/MultiUserTests.java

```diff
@@ -120,6 +120,16 @@ public void grantAndRevokeNestedDirWriteAccess() {
         PeergosNetworkUtils.grantAndRevokeNestedDirWriteAccess(network, random);
     }
 
+    @Test
+    public void grantAndRevokeNestedDirWriteAccess2() {
```

Comment:
Better method name

Reply:
I agree, but couldn't think of one. Any ideas?

Reply:
done